extensions/README.md

```diff
@@ -45,45 +45,45 @@
       - [Filesystem](common/vault/filesystem-vault/)
       - [HashiCorp](common/vault/hashicorp-vault/)
 - Control Plane
-  - [Data Management API](control-plane/api/data-management/)
-    - [Configuration](control-plane/api/data-management/api-configuration/)
-    - [Asset](control-plane/api/data-management/asset-api/)
-    - [Catalog](control-plane/api/data-management/catalog-api/)
-    - [Contract Agreement](control-plane/api/data-management/contractagreement-api/)
-    - [Contract Definition](control-plane/api/data-management/contractdefinition-api/)
-    - [Contract Negotiation](control-plane/api/data-management/contractnegotiation-api/)
-    - [Policy Definition](control-plane/api/data-management/policydefinition-api/)
-    - [Transfer Process](control-plane/api/data-management/transferprocess-api/)
+  - [Data Management API](control-plane/api/data-management-api/)
+    - [Configuration](control-plane/api/data-management-api/data-management-api-configuration/)
+    - [Asset](control-plane/api/data-management-api/asset-api/)
+    - [Catalog](control-plane/api/data-management-api/catalog-api/)
+    - [Contract Agreement](control-plane/api/data-management-api/contract-agreement-api/)
+    - [Contract Definition](control-plane/api/data-management-api/contract-definition-api/)
+    - [Contract Negotiation](control-plane/api/data-management-api/contract-negotiation-api/)
+    - [Policy Definition](control-plane/api/data-management-api/policy-definition-api/)
+    - [Transfer Process](control-plane/api/data-management-api/transfer-process-api/)
   - [Data Plane Transfer](control-plane/data-plane-transfer/)
   - [HTTP Receiver](control-plane/http-receiver/)
   - Provision
     - [Blob](control-plane/provision/blob-provision/)
-    - [HTTP](control-plane/provision/http-provisioner/)
+    - [HTTP](control-plane/provision/http-provision/)
     - [S3](control-plane/provision/s3-provision/)
   - Store
     - Cosmos
-      - [Asset Index](control-plane/store/cosmos/assetindex-cosmos/)
+      - [Asset Index](control-plane/store/cosmos/asset-index-store-cosmos/)
```

Comment:
Do we need the *store* here? `AssetIndex` is already the name of the store, as opposed to the others that actually have *store* in the interface name. Same for the other asset index modules.